gprofiler/utils.py

```diff
@@ -330,6 +332,10 @@ def _switch_and_run():
 
 
 def log_system_info():
```

Comment:
The function name doesn't make sense anymore, it does more then just logging

Reply:
Yeah I was just lazy not splitting it off. Will do